tests: turn off watchdog for sam_e70 test kernel/critical

maksimmasalski opened the pull request:
Move #20607 to the master instead of adding that fix to v1.14-branch.
Wrong configuration of a watchdog for sam_e70_xplained_board caused
endless reboot loop. During tests I found out that if to decrease
NUM_MILLISECONDS numeric constant from 5000ms to 4000ms, then
test will complete successfully. It seems that problem caused by
a watchdog.

Signed-off-by: Maksim Masalski <maksim.masalski@intel.com>

nashif reviewed (changes requested):
Why do you need this? This is already fixed in tests/kernel/critical/testcase.yaml

maksimmasalski commented:
Referenced fix in #20607 for v1.14-branch. That is closed, because master already had that fix.

Commit:
tests: off watchdog for sam_e70 test kernel/critical
Move #20607 to the master instead of adding that fix to v1.14-branch.
Wrong configuration of a watchdog for sam_e70_xplained_board caused
endless reboot loop. During tests I found out that if to decrease
NUM_MILLISECONDS numeric constant from 5000ms to 4000ms, then
test will complete successfully. It seems that problem caused by
a watchdog.

Signed-off-by: Maksim Masalski <maksim.masalski@intel.com>
